streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/HighAvailabilityTaskAssignorTest.java

```diff
@@ -88,11 +90,9 @@ private void createTaskAssignor() {
 
     @Test
     public void shouldDecidePreviousAssignmentIsInvalidIfThereAreUnassignedActiveTasks() {
-        client1 = EasyMock.createNiceMock(ClientState.class);
-        expect(client1.prevActiveTasks()).andReturn(singleton(TASK_0_0));
-        expect(client1.prevStandbyTasks()).andStubReturn(EMPTY_TASKS);
-        replay(client1);
-        allTasks =  mkSet(TASK_0_0, TASK_0_1);
+        client1 = new ClientState();
```

Comment:
I converted all these tests to stop mocking ClientState and just instantiate it properly instead. The problem was that as soon as I changed the HATA code, it needed different invocations and a bunch of these tests got NPEs.